JsonConverter.bas

```diff
@@ -117,22 +117,22 @@ End Type
 #Else
 
 Private Type utc_SYSTEMTIME
-    utc_wYear As Integer
-    utc_wMonth As Integer
-    utc_wDayOfWeek As Integer
-    utc_wDay As Integer
-    utc_wHour As Integer
-    utc_wMinute As Integer
-    utc_wSecond As Integer
-    utc_wMilliseconds As Integer
+    utc_wYear As Long
+    utc_wMonth As Long
+    utc_wDayOfWeek As Long
+    utc_wDay As Long
+    utc_wHour As Long
+    utc_wMinute As Long
+    utc_wSecond As Long
+    utc_wMilliseconds As Long
 End Type
 
 Private Type utc_TIME_ZONE_INFORMATION
     utc_Bias As Long
-    utc_StandardName(0 To 31) As Integer
+    utc_StandardName(0 To 31) As Long
     utc_StandardDate As utc_SYSTEMTIME
     utc_StandardBias As Long
-    utc_DaylightName(0 To 31) As Integer
+    utc_DaylightName(0 To 31) As Long
```

Comment:
These must stay as `Integer` types; changing them to `Long` will break them and they will not contain the correct data.